server/protocols/webadmin/webadmin-mailbox-deleted-message-vault/src/main/java/org/apache/james/webadmin/vault/routes/ExportService.java

```diff
@@ -81,7 +90,16 @@ private BlobId zipToBlob(User user, Flux<DeletedMessage> messages) throws IOExce
     }
 
     private DeletedMessageZipper.DeletedMessageContentLoader contentLoader(User user) {
-        return message -> Mono.from(vault.loadMimeMessage(user, message.getMessageId())).blockOptional();
+        return message -> Mono.from(vault.loadMimeMessage(user, message.getMessageId()))
+            .onErrorResume(CONTENT_NOT_FOUND_PREDICATE, throwable -> {
+                LOGGER.error(
```

Comment:
This should be an info in my opinion. Rational: we expect these deletion being a bit too greedy and exports skipping related messages.

This is not something a DEV or an admin needs to be freaked out about.